### ISSUE TITLE
Add troubleshooting step: logs --all-containers

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn-contrib/job-executor-service#123]
During trying out job-executor I ended up getting this error in Bridge:
```
Job job-executor-service-job-855e19ed-baed-48b3-973e-0e62-1 failed: job job-executor-service-job-855e19ed-baed-48b3-973e-0e62-1 failed. Reason: BackoffLimitExceeded, Message: Job has reached the specified backoff limit
```

Unfortunately this does not tell me why the job has failed, so I had to start troubleshooting on my own.
A very handy troubleshooting step is to look at Kubernetes logs, especially the log of the job or the pod(s).

For this the following steps need to be done:
```console
$ kubectl -n keptn get pods
...
job-executor-service-799fc875c7-hjbjw                           2/2     Running      0          19h
job-executor-service-job-855e19ed-baed-48b3-973e-0e62-1-qdswm   0/1     Init:Error   0          18m
job-executor-service-job-f5f44383-054d-48d0-b219-e2d6-1-p7bbl   0/1     Error        0          19h
job-executor-service-job-ff152cae-1238-4249-b5e3-67fd-1-h2vmc   0/1     Init:Error   0          24m
...
```

Select the pod that you want to inspect, e.g. the most recent one: `job-executor-service-job-855e19ed-baed-48b3-973e-0e62-1-qdswm`

```console
$ kubectl -n keptn logs job-executor-service-job-855e19ed-baed-48b3-973e-0e62-1-qdswm --all-containers
```

And look at the output, e.g.:

```
kubectl -n keptn logs job-executor-service-job-855e19ed-baed-48b3-973e-0e62-1-qdswm -c init-job-executor-service-job-855e19ed-baed-48b3-973e-0e62-1
2021/12/17 10:31:18 Error while copying files: could not find file or directory jmeter/basiccheck.jmx for task Run jmeter smoke tests
```

# Definition of Done

- [ ] Add a troubleshooting section with this as the first troubleshooting step

**Comment by grabnerandi** [comment]
I just ran into the same issue and it took me a while to do some troubleshooting
Would be great to provide those types of messages to the end user as the fix is simple - but - getting to the actual reason is not

**Comment by christian-kreuzberger-dtx** [comment]
This is also addressed using #214, so we don't need to add it to the troubleshooting section.
